lib/glimesh_web/templates/about/streaming.html.eex

```diff
@@ -0,0 +1,118 @@
+<div class="jumbotron jumbotron-fluid jumbotron-glimesh d-flex align-items-center">
+    <div class="container">
+        <h1 class="display-4 text-center"><%= gettext("Streaming on Glimesh") %></h1>
+    </div>
+</div>
+<h2><%= gettext("Features & Focuses") %></h2>
+<p>
+    <%= gettext("At Glimesh we value an excellent experience while streaming. So we've taken a great deal of consideration into how and why we are designing features to make you and your viewers experience an excellent one. Let's take a brief look at what makes us stand out in the crowd.") %>
+</p>
+
+<div class="row">
+    <div class="col-md-4">
+        <div class="card h-100">
+            <div class="card-body">
+                <h4 class="text-center mb-4"><%= gettext("Discoverability") %></h4>
+                <p>
+                    <%= gettext("Discoverability is a main focus of our platform. During our initial launch it will be extremely easy to find new content to watch. However as we grow this becomes more of a problem, and we want to experiment with a couple of different options for making it easy to find new and smaller streams. From the very beginning we are working hard on making streaming equitable for all, so you don’t have to compete with unfair advantages.") %>
```

Comment:
```suggestion
                    <%= gettext("Discoverability is a primary focus of our platform. During our initial launch, it will be straightforward to find new content to watch. However, as we grow, this becomes more of a problem, and we want to experiment with a couple of different options for making it easy to find new and smaller streams. From the very beginning, we are working hard on making streaming equitable for all, so you don’t have to compete with unfair advantages.") %>
```